app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -46,18 +46,56 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       t,
       :items => [
         button(
-          :middleware_server_reload,
```

Comment:
no. Moved further down to have the operations in the same order I saw on another provider (forgot which one).